policyengine_us/variables/gov/states/ar/tax/income/deductions/itemized/ar_itemized_deductions.py

```diff
@@ -0,0 +1,53 @@
+from policyengine_us.model_api import *
+
+
+class ar_itemized_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona itemized deductions"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=21"
+    defined_for = StateCode.AR
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(period).gov.irs.deductions.itemized
+        less_salt_deds = tax_unit("itemized_deductions_less_salt", period)
+
+        agi = tax_unit("adjusted_gross_income", period)
+        spouse_agi = tax_unit("spouse_separate_adjusted_gross_income", period)
+
+        # Real estate tax + Personal property tax
+        person_real_estate_deds = person("real_estate_taxes", period)
```

Comment:
for the msc deduction: 
we need to cap it at this parameter: (0.2)
/policyengine_us/parameters/gov/irs/deductions/itemized/misc/floor.yaml
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/1692a4cb-631a-453f-be5d-82aaf4097a54)

Reply:
Thank you for the comment, Pavel. In the instruction for Arkansas itemized deductions, the limit ratio is 2%, which is the same as the value of the federal floor, do I need to make another parameter for this threshold as well? Thank you!